libs/init_runtime/include/hpx/hpx_init_params.hpp

```diff
@@ -51,6 +51,17 @@ namespace hpx {
     }    // namespace resource
     /// \endcond
 
+    enum runtime_type
```

Comment:
I would like to see our enums in APIs to be slowly migrated to `enum class` types. This would also homogenize names to something like `runtime_type::local`

Reply:
Agreed.